### PR TITLE
logo + flyer redesign

### PR DESCRIPTION
**Pull request by ttraenkler**
this integrates from @pranjal9599 forks and intends to solve to #9 #4

- draft of logo and layout proposal @ashleygwilliams 
- design idea (@pranjal9599)
- refinement of logo and design + layout fine tunings (@ttraenkler)
- buttons / links functional
- mobile layout refinements

### CODE REVIEW

**Comment by ttraenkler** [comment]
@torch2424 @ashleygwilliams I've added the missing links, refined the logo and layout, fixed mobile view etc. Thanks @pranjal9599 for a great composition.

File: pages/index.tsx [hunk 1]
```diff
@@ -1,123 +1,205 @@
...
 
-const ContactUs = styled.div`
-  color: #FFFFFF;
+const Footer = styled.footer`
   text-align: center;
+  color: rgba(255, 255, 255, 0.7);
+  margin: 2vh 0;
+  font-size: 1.1rem;
 
-  a, a:visited {
```
Comment (torch2424) [inline pages/index.tsx]: Curious to why you aren't applying styles to a:visited as well? From my testing, visiting links can trigger the default pirple color, which then breaks the design. 🤔

Comment (ttraenkler) [reply pages/index.tsx]: Sure, let me review the CSS as well. So far I focused on the problems I could spot looking at the picture and resizing the width

Comment (ttraenkler) [reply pages/index.tsx]: Sure, let me review the CSS as well. So far I focused on the problems I could spot looking at the picture and resizing the width

Comment (ttraenkler) [reply pages/index.tsx]: fixed, please approve :-) i haven't reviewed everything but we can always reiterate

**Pull request merged**